test/common_tabular_tests.py

```diff
@@ -796,13 +807,22 @@ def verify_explain_model_shap_values_multiclass(self, shap_values_output=ShapVal
         explanation = explainer.explain_global(x_test)
         is_probability = shap_values_output != ShapValuesOutput.DEFAULT
         self.validate_explanation(explanation, is_multiclass=True, is_probability=is_probability)
-        # validate explanation has init_data on it
-        assert(explanation.init_data is not None)
+        # validate explanation has init_data on it in mimic explainer case (note there is none for TreeExplainer)
+        if hasattr(explainer, SURROGATE_MODEL):
+            assert(explanation.init_data is not None)
 
-    def verify_explain_model_shap_values_binary(self, shap_values_output=ShapValuesOutput.DEFAULT):
+    def verify_explain_model_shap_values_binary(self, shap_values_output=ShapValuesOutput.DEFAULT,
+                                                use_xgboost=False, use_tree=False):
```

Comment:
I recommend making `use_xgboost=False, use_tree=False` an enum to make it easier to add models in the future

Reply:
done!